### DISCUSSION TITLE
"Expand/Collapse all" Headings

### DISCUSSION THREAD

**Discussion by leandromartinez22** [outline/outline#7276] [Ideas]
It would be good to have an "Expand all" and "Collapse all" button/toggle at the top of every page.

It is great for visibility to have all the headings collapsed (specially when dealing with pages that contain lots of information) but if you are trying to look for a specific word in the page (using CTRL + F), you can't find the word if it's hidden in a collapsed heading and it's really tedious to manually expand every heading to then search for the word, and then collapse them all back again.

**Comment by tommoor** [comment]
Well, there is probably a need to "pop open" collapsed heading if there is a search result within the collapsed area too

**Comment by Codinator007** [comment]
We would love to have exactly this feature. Collapse and Expand all Headings within a document.

**Reply by tommoor** [reply]
Is there a common shortcut for this?
